flink-end-to-end-tests/test-scripts/test_resume_savepoint.sh

```diff
@@ -17,11 +17,25 @@
 # limitations under the License.
 ################################################################################
 
+if [ -z $1 ] || [ -z $2 ]; then
+  echo "Usage: ./test_resume_savepoint.sh <original_dop> <new_dop>"
+  exit 1
+fi
+
 source "$(dirname "$0")"/common.sh
 
-# modify configuration to have 2 slots
+ORIGINAL_DOP=$1
+NEW_DOP=$2
+
+if (( $ORIGINAL_DOP >= $NEW_DOP )); then
+  NUM_SLOTS=$(( $ORIGINAL_DOP + 1 ))
```

Comment:
would be good to explain where the +1 comes from. (i guess it is for the kafka job)

Reply:
👌 Yes, it is for the Kafka event generator job.